src/contractTest/java/uk/gov/hmcts/reform/wataskmonitor/consumer/ccd/StartEventForCaseWorkerConsumerTest.java

```diff
@@ -1,109 +0,0 @@
-package uk.gov.hmcts.reform.wataskmonitor.consumer.ccd;
-
-import au.com.dius.pact.consumer.dsl.PactDslWithProvider;
-import au.com.dius.pact.consumer.junit5.PactTestFor;
-import au.com.dius.pact.core.model.RequestResponsePact;
-import au.com.dius.pact.core.model.annotations.Pact;
-import org.json.JSONException;
-import org.junit.jupiter.api.Test;
-import uk.gov.hmcts.reform.ccd.client.model.CaseDataContent;
-import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
-import uk.gov.hmcts.reform.ccd.client.model.StartEventResponse;
-import uk.gov.hmcts.reform.wataskmonitor.consumer.ccd.util.CcdConsumerTestBase;
-
-import java.util.Map;
-
-import static org.hamcrest.CoreMatchers.is;
-import static org.hamcrest.MatcherAssert.assertThat;
-import static org.junit.Assert.assertNotNull;
-import static org.springframework.http.HttpHeaders.AUTHORIZATION;
-import static org.springframework.http.HttpHeaders.CONTENT_TYPE;
-import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
-import static uk.gov.hmcts.reform.wataskmonitor.consumer.ccd.util.PactDslBuilderForCaseDetailsList.buildStartEventForCaseWorkerPactDsl;
-
-public class StartEventForCaseWorkerConsumerTest extends CcdConsumerTestBase {
-
-    public Map<String, Object> setUpStateMapForProviderWithCaseData(CaseDataContent caseDataContent)
-        throws JSONException {
-        Map<String, Object> caseDataContentMap = super.setUpStateMapForProviderWithCaseData(caseDataContent);
-        caseDataContentMap.put(EVENT_ID, START_APPEAL);
-        return caseDataContentMap;
-    }
-
-    @Pact(provider = "ccdDataStoreAPI_Cases", consumer = "wa_task_monitor")
-    public RequestResponsePact startEventForCaseWorker(PactDslWithProvider builder) throws JSONException {
-        return builder
-            .given("A Start Event for a Caseworker is  requested",
-                setUpStateMapForProviderWithCaseData(caseDataContent)
-            )
-            .uponReceiving("A Start Event for a Caseworker")
-            .path(buildPath())
-            .method("GET")
-            .matchHeader(AUTHORIZATION, AUTH_TOKEN)
-            .matchHeader(SERVICE_AUTHORIZATION, SERVICE_AUTH_TOKEN)
-            .matchHeader(CONTENT_TYPE, APPLICATION_JSON_VALUE)
-            .willRespondWith()
-            .matchHeader(CONTENT_TYPE, APPLICATION_JSON_VALUE)
-            .status(200)
-            .body(buildStartEventForCaseWorkerPactDsl(START_APPEAL))
-            .toPact();
-    }
-
-    @Test
-    @PactTestFor(pactMethod = "startEventForCaseWorker")
-    public void verifyStartEventForCaseworker() throws JSONException {
-
-        final StartEventResponse startEventResponse = coreCaseDataApi.startEventForCaseWorker(
```

Comment:
This is used currently in CaseManagementDataService

        StartEventResponse startEventResponse = coreCaseDataApi.startEventForCaseWorker(
            parameter.getUserAuthorization(),
            parameter.getServiceAuthorization(),
            parameter.getUserId(),
            "IA",
            "Asylum",
            parameter.getCaseId(),
            EVENT_ID
        );

This pact test needs to be reintroduced.